fixity/fixity.py

```diff
@@ -48,6 +48,8 @@ def fetch_environment_variables(namespace):
     namespace.ss_url = _get_environment_variable('STORAGE_SERVICE_URL')
     if not namespace.ss_url.endswith('/'):
         namespace.ss_url = namespace.ss_url + '/'
+    namespace.ss_user = _get_environment_variable('STORAGE_SERVICE_USER')
+    namespace.ss_key = _get_environment_variable('STORAGE_SERVICE_KEY')
```

Comment:
Should these be optional, like REPORT_USERNAME/REPORT_PASSWORD, to handle older storage service versions?